fuzzers/000-init-db/Makefile

```diff
@@ -22,6 +22,8 @@ SEGBITS=\
       dsp \
       hclk \
       int \
+      ioi \
```

Comment:
This is not longer correct, and lioi3 and rioi3 files do not get initialized to empty.  Need to update.

Reply:
Done